summary_workflows/identification/I2_benchmark/I2_benchmark_specification.md

```diff
@@ -47,45 +64,146 @@ Fields:
 - **score** - not used, leave as "."
 - **strand** - defines the strand; either "." (=no strand) or "+" or "-".
 
-#### Format 2
+#### Input 2
 
 This BED file contains single-nucleotide position of poly(A) sites identified from the orthogonal 3'end-seq dataset.  
-Fields are the same as in Format 1.
+Fields are the same as in Input 1.
```

Comment:
Should the PAS databases also be declared as input? Or maybe mention that they need to have the same format as the orthogonal 3'end seq data, or whatever else makes sense here

Reply:
Correct, databases are not used (for now).